test/e2e/storage/drivers/in_tree.go

```diff
@@ -1376,6 +1376,8 @@ func (g *gcePdDriver) CreateVolume(config *storageframework.PerTestConfig, volTy
 			},
 		}
 	}
+	framework.Logf("volume type %q: config.ClientNodeSelection.Selector = %v", volType, config.ClientNodeSelection.Selector)
```

Comment:
Is this supposed to be in the PR?

It looks like temporary debug output.

Reply:
As it's a test I figured the information was useful.